src/features/common/Layout/UserLayout/UserLayout.tsx

```diff
@@ -492,7 +527,7 @@ function UserLayout(props: any): ReactElement {
             className={styles.navlink}
             //logout user
             onClick={() => {
-              localStorage.removeItem('impersonatedEmail');
+              localStorage.removeItem('impersonatedData');
```

Comment:
Be consistent with naming. Here you should remove `impersonationData` not `impersonatedData`

Reply:
This is missed out.